.travis.yml

```diff
@@ -2,20 +2,20 @@ sudo: required
 language: c
 compiler: gcc
 env:
-- TARGET=x86_64-linux-gnu
-- TARGET=x86-linux-gnu
-- TARGET=arm-linux-gnueabihf
-- TARGET=aarch64-linux-gnu
-- TARGET=mipsel-unknown-linux-gnu
+- HOST=x86_64-linux-gnu
+- HOST=x86-linux-gnu
+- HOST=arm-linux-gnueabihf
+- HOST=aarch64-linux-gnu
+- HOST=mipsel-linux-gnu
 # Currently experiencing build failures here
-#- TARGET=powerpc64-linux-gnu
+#- HOST=powerpc64-linux-gnu
 
 linux-s390x: &linux-s390x
   os: linux
   arch: s390x
-  env: TARGET=s390x-linux-gnu
+  env: BUILD=s390x-linux-gnu HOST=s390x-linux-gnu
   script:
-    - ./autogen.sh
+    - autoreconf -i
     - ./configure
```

Comment:
when we run ./autogen.sh, it does autoreconf then run configure. so it was redundant. in cross compilation case (below), on top of running configure twice, it gives us confusing logs.

Reply:
Yea we can probably dump autogen.sh, I never use it, anyway.